EIPS/eip-2535.md

```diff
@@ -3,37 +3,37 @@ eip: 2535
 title: Diamonds, Multi-Facet Proxy
 author: Nick Mudge (@mudgen)
 discussions-to: https://github.com/ethereum/EIPs/issues/2535
-status: Last Call
-last-call-deadline: 2020-08-15
+status: Review
+description: A standard for creating modular smart contract systems that can be extended after deployment
 type: Standards Track
 category: ERC
 created: 2020-02-22
+requires: 1538, 2535
```

Comment:
1538 is not required, please remove that.

Reply:
It's mentioned in the EIP. Therefore, it goes in the requires list.

Reply:
I recommend you remove the reference then.  EIPs are for standardization, and they shouldn't be used as history, tutorials, documentation, etc. If understanding EIP-1538 isn't required to implement this standard, then this standard shouldn't be referring to it.

Generally speaking, the discussions-to link is a good place to put supplemental information and links that may be useful to future readers, but isn't strictly required for standardization purposes.

Reply:
Yes, I mean EIP-1538.

Reply:
I *think* we agreed that since Withdrawn is a terminal status, it can be referenced from other EIPs.  Effectively, anything that is final or withdrawn is terminal and thus can be referenced freely.